### PR TITLE
add deleted tags validation

### PR DESCRIPTION
**Pull request by dmitriyfolio**

### CODE REVIEW

File: cypress/integration/quickmark/instance-quickmark.spec.js [hunk 1]
```diff
@@ -105,24 +101,23 @@ describe('Manage inventory Bib records with quickMarc editor', () => {
...
-        const expectedUpdatedValue = QuickMarcEditor.updateExistingField();
+        QuickMarcEditor.deletePenaltField().then(deletedTag => {
+          const expectedUpdatedValue = QuickMarcEditor.updateExistingField();
 
-        QuickMarcEditor.pressSaveAndClose();
-        QuickMarcEditor.deleteConfirmationPresented();
-        QuickMarcEditor.confirmDelete();
+          QuickMarcEditor.pressSaveAndClose();
+          QuickMarcEditor.deleteConfirmationPresented();
+          QuickMarcEditor.confirmDelete();
```
Comment (MaksimTerekhov) [inline cypress/integration/quickmark/instance-quickmark.spec.js:109]:
```
QuickMarcEditor.pressSaveAndClose();
QuickMarcEditor.deleteConfirmationPresented();
QuickMarcEditor.confirmDelete();
```
Colud it be inside one function? Seems you need to reuse it several times

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import QuickMarcEditorFragment from '../quickMarcEditor';
 import inventoryActions from './inventoryActions';
 import InventoryInstanceEdit from './InventoryInstanceEdit';
 import HoldingsRecordView from './holdingsRecordView';
+import inventoryViewSource from './inventoryViewSource';
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/inventoryInstance.js:6]:
```
import inventoryActions 
import InventoryInstanceEdit 
import HoldingsRecordView 
import inventoryViewSource 
```
Please use first capital letter in imports

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 2 above)
Comment (dmitriyfolio) [reply cypress/support/fragments/inventory/inventoryInstance.js:6]: fixed

File: cypress/integration/quickmark/instance-quickmark.spec.js (hunk 1 above)
Comment (dmitriyfolio) [reply cypress/integration/quickmark/instance-quickmark.spec.js:109]: additional confirmation dialog related with case when user removed feilds in Marc editor. These steps can be useful in tests level. I've added TODO related with these tests redesign.

**Review by KyryloBrener — APPROVED** [review]

**Pull request merged**